tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage1/requirements_torch_cpu.txt

```diff
@@ -1,2 +1,2 @@
-torch==1.10.0
+torch==1.11.0
```

Comment:
Is it possible to avoid making changes on this file? Inference pipelines usually do not need pytorch.  As you can see now, updating pytorch needs a lot of extra work.